code-challenges.test.js

```diff
@@ -1,4 +1,11 @@
-// ASSESSMENT 2: Coding practical questions with Jest
+// // ASSESSMENT 2: Coding practical questions with Jest
+
+const { first } = require("lodash")
```

Comment:
Using other libraries is perfectly fine. But an even better exercise would be to see if you can write a function that does that behavior you want and use that

Reply:
Looks like you didn't end of using lodash after all haha  Keep in mind that requiring packages uses up memory and resources so it's good to get in the habit of cleaning up imports/requires statements if it's unused